docs/make.jl

```diff
@@ -2,12 +2,14 @@ using Documenter, AbstractFFTs
 
 makedocs(
     modules = [AbstractFFTs],
+    repo="https://github.com/JuliaMath/AbstractFFTs.jl/blob/{commit}{path}#{line}",
```

Comment:
Why this? Isn't
```suggestion
    repo="JuliaMath/AbstractFFTs.jl",
```
sufficient?